app/api/endpoints/tag_sets.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+# Controls API V1 {::Core::Endpoint::Base endpoints} for TagGroups
+class Endpoints::TagSets < Core::Endpoint::Base
+  model {}
+
+  instance {}
+end
```

Comment:
Was this necessary for some reason? We shouldn't be adding new V1 endpoints - Stuart's project was to replace remaining V1 use with V2 API.

Reply:
I was not sure about this and I just copied how it was for other end points. :D It sounds like it is for supporting V1. Shall I remove this?

Reply:
Yes pretty sure this is just for V1 so unnecessary.